Lib/concurrent/futures/_base.py

```diff
@@ -533,6 +533,49 @@ def set_exception(self, exception):
             self._condition.notify_all()
         self._invoke_callbacks()
 
+    def then(self, fn, executor):
+        """Calls the function fn, which takes only the future as an argument,
```

Comment:
Python uses a different style for docstrings.

Reply:
I tried to use the one that already existed in the concurrent.futures package for consistency.  I thought it was weird too.